src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -963,6 +964,9 @@ well as menu structures (for main menu and popup menus).
    <cmd id="reflowComment"
         menuLabel="Reflow Co_mment"
         desc="Reflow selected comment lines so they wrap evenly"/>
+   <cmd id="renameInFile"
```

Comment:
We should include the command both in the menubar and within the code tools menu. For both contexts I think it should be right in the same group as Extract Function and Extract Variable.